src/services/userService.js

```diff
@@ -0,0 +1,34 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const newMessageService = (newMessage, user, emiter) => {
+  const unicId = uuidv4();
```

Comment:
There is a typo in the variable name `unicId`. It should be `uniqueId` for better readability and accuracy.